commands/http/handler.go

```diff
@@ -182,6 +174,11 @@ func sendResponse(w http.ResponseWriter, req cmds.Request, res cmds.Response) {
 		h.Set(contentLengthHeader, strconv.FormatUint(res.Length(), 10))
 	}
 
+	if _, ok := res.Output().(io.Reader); ok {
+		mime = ""
```

Comment:
preserve the comment, else we won't know why the hell we're setting `mime = ""` in a few weeks.